src/components/MetaData/index.tsx

```diff
@@ -96,6 +96,8 @@ interface Props {
       metadata_description?: string;
       metadata_title?: string;
     };
+    canonical_website?: string | null;
```

Comment:
`canonical_website` isn't used anywhere do we still need it defined?

Reply:
updated